docs/README.md

```
@@ -119,4 +292,4 @@ When contributing to the project:
 
 ---
 
-*Last updated: June 2025 - Environment refactoring complete and deployed*
+*Last updated: September 2025 - Benchmark runner and batch API added*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)